doc/10-icinga-template-library.md

```diff
@@ -5589,11 +5589,11 @@ ssl_cert_ignore_ocsp          | **Optional.** Do not check revocation with OCSP.
 
 #### jmx4perl <a id="plugin-contrib-command-jmx4perl"></a>
 
-The [check_jmx4perl](http://search.cpan.org/~roland/jmx4perl/scripts/check_jmx4perl) plugin
+The [check_jmx4perl](https://search.cpan.org/~roland/jmx4perl/scripts/check_jmx4perl) plugin
```

Comment:
Redirects to https://metacpan.org/pod/distribution/jmx4perl/scripts/check_jmx4perl, update as well?